celery/app/trace.py

```diff
@@ -141,6 +141,11 @@ def get_log_policy(task, einfo, exc):
             return log_policy_expected
         return log_policy_unexpected
 
+    
+def get_task_name(name, request):
```

Comment:
Thanks for applying the changes! Sorry for not suggesting it correctly before, but the repetition of `name` in the function name - signature can possibly be omitted:

```
def get_task_name(request, default):
```

What do you think?

Reply:
It's fine with me.